### PR TITLE
Virtual LP FIxups to Support ethernet connector

### PR DESCRIPTION
**Pull request by callendorph**
1.  Support `build-all` overrides better
2.  Tools for easier child node lookup by name
3.  Bugfix for `get-pads` that fixes the courtyard outline for LPs with multiple child nodes
4.  Added silkscreen from package body helper routine.

### CODE REVIEW

File: src/landpatterns/packages.stanza [hunk 1]
```diff
@@ -470,6 +470,17 @@ public defmulti build-all (
   virt:VirtualLP
   ) -> False
 
+public defn package-build-all-seq (
```
Comment (emspin) [inline src/landpatterns/packages.stanza:473]: qq: what's the `seq` suffix here referring to?

Comment (callendorph) [reply src/landpatterns/packages.stanza:473]: Sequence - ie, these functions all get called in a particular order. 

I could replace it with `order` if you think that would be less confusing (ie, not confused with list operator `seq`)

Comment (emspin) [reply src/landpatterns/packages.stanza:473]: Yeah, that's probably preferred. I had assumed it referred to the order in which things were being built, as opposed to the operator, but explicitly dodging the possible ambiguity is better.

**Review by emspin — APPROVED** [review]
Only comment is to rename function to not clash with term `seq`

Thanks for making the changes!

**Pull request merged**